internal/controller/helmchart_controller.go

```diff
@@ -884,6 +888,9 @@ func (r *HelmChartReconciler) reconcileArtifact(ctx context.Context, _ *patch.Se
 	// Record it on the object
 	obj.Status.Artifact = artifact.DeepCopy()
 	obj.Status.ObservedChartName = b.Name
+	if obj.Spec.IgnoreMissingValuesFiles {
```

Comment:
otherwise we should clear it to avoid stale values.

Reply:
Updated.